### ISSUE TITLE
error displayed on dynnet configuration

### ISSUE THREAD

**Issue by mschaaf** [jaor/xmobar#385]
I get from time to time the following error in xmobar
![2019-03-28-135839_1041x22_scrot](https://user-images.githubusercontent.com/703355/55159674-e8261c00-5161-11e9-9943-3f3f02196765.png)

It seems it tries to find a virtual device created by docker which is no longer available. I never saw xmobar to choose another device after this happens. Is their a way to configure xmobar to ignore these devices?

my config:
```
                    , Run DynNetwork [
                          "--template" , "<dev>: <tx>kB/s|<rx>kB/s",
                          "--Low"      , "1000",
                          "--High"     , "5000",
                          "--low"      , "#1f2022",
                          "--normal"   , "#ffa500",
                          "--high"     , "#F2241F"
                        ] 10

```
xmobar version: 0.24.5

**Comment by jaor** [comment]
Hmm, no, there's currently no option for that.  Maybe we could add one,
perhaps as a regular expression to be excluded or as a list of allowed
interfaces.  Another option could be to recover gracefully from those
errors, and try the next one... actually, we recover gracefully from
similar errors (for instance, i can just disable my network altogether
and xmobar will pick it up when it comes back), so i guess i should
check that first.

**Comment by liskin** [comment]
This is actually already fixed in 0.26 by 3b88faefe7a2a512ebf2dcde4f3cdb3e89b98393, and the support for filtering was merged last week as well: e9638fdc41ccd0876f17b4344461deaa109332e2. I think this can be closed.

**Comment by mschaaf** [comment]
Thanks for the reminder. I will try the latest version.
